SUMMARY.md

```diff
@@ -41,6 +41,7 @@
       * [Milestones](getting-started/modules/core/milestones.md)
       * [Plugins](getting-started/modules/core/plugin.md)
       * [Outbox](getting-started/modules/core/outbox.md)
+      * [Outbox RabbitMQ](getting-started/modules/core/outbox-rabbitmq.md)
```

Comment:
Maybe indent this by one so it forms a new group? Or we wait until there's more implementations for the outbox. What do you think?

Reply:
done